src/tetris/field/field.ts

```diff
@@ -39,22 +40,31 @@ export default class Field {
 	//region public methods
 	public update(time: number, delta: number): void {
 		if (!this.activeBrick) {
-			this.activeBrick = this._brickFactory.newBrick(this);
+			this._generateNewBrick(time);
 		} else {
-			this._activeBrick.update(time, delta);
+			if (this._nextActiveBrickDrop < time) {
+				this._nextActiveBrickDrop = time + this._activeBrickDropInterval;
+				this.activeBrick.dropOne();
+			}
+			this.activeBrick.update(time, delta);
 		}
 		if (this.activeBrick.isStuck()) {
-			this._addToField(this._activeBrick.blocks);
+			this._addToField(this.activeBrick.blocks, this.activeBrick.position);
 			this.addBrick(this.activeBrick);
-			this.activeBrick = null;
+			this._generateNewBrick(time);
```

Comment:
Do we really want to create a new brick at this point? If so, it would be (theoretically) possible to create two new bricks in just one frame.
I would suggest to cut this line.